python/whylogs/core/schema.py

```diff
@@ -61,9 +61,12 @@ class DatasetSchema:
     cache_size: int = 1024
     schema_based_automerge: bool = False
 
-    def __init__(self) -> None:
+    def __init__(self, config: MetricConfig = None) -> None:
```

Comment:
```suggestion
    def __init__(self, config: Optional[MetricConfig] = None) -> None:
```

Also we should consider other params here as well (not a blocking comment)

Reply:
Fixed